src/main/bin/ccm-initialise

```diff
@@ -128,9 +127,9 @@ MakeCacheRoot($this_app, $cache_root, $dopts);
 
 # make global lock file (mask and GID take care of the permissions)
 $this_app->verbose("Creating lockfile");
-open TMP, ">$cache_root/global.lock";
-print TMP "no\n";
-close TMP;
+open my $TMP, ">$cache_root/global.lock";
```

Comment:
i have no clue. the critic test currently only fails on limited set of policies (https://github.com/quattor/maven-tools/blob/master/build-scripts/src/main/perl/Test/Quattor/Critic.pm#L22)

feel free to have a look at the jenkins output (eg https://jenkins1.ugent.be/job/maven-tools-pr-builder/144/org.quattor.maven$build-scripts/consoleFull) and nominate some more policies to fail on.